src/types/apps-rest-types.ts

```diff
@@ -0,0 +1,50 @@
+/**
```

Comment:
This file should be named `apps-auth-rest-types.ts`, right?

Reply:
@aecorredor I think Apps is more appropriate if we ever wanted to add anything else under this origin `https://apps.us.lifeomic.com`, not just the `/auth` routes